docs/docs/upgrade/index.md

```diff
@@ -24,6 +24,113 @@ Upgrading to a newer version of Marathon should be executed in the following ord
 1. Install the new version of Marathon on all remaining nodes with the old version.
 1. Start all other instances of Marathon to build a quorum.
 
+## Releases
+
+It is recommended not to skip Marathon releases when upgrading Marathon. For instance, if upgrading to 1.6.x from 1.4.x, you should upgrade to 1.5.x first, and only then to 1.6.x.
+
+# Upgrading to 1.6
+
+Release Notes: https://github.com/mesosphere/marathon/releases/tag/v1.6.352
+
+The recommended Mesos version is 1.5.0 or later.
+
+## Breaking changes
+
+- **SentryAppender**.
+The Sentry Raven log appender has been updated to version 8.0.x. Users that have enabled the Sentry Raven appender will need to update their configuration according to the [sentry migration guide](https://docs.sentry.io/clients/java/migration/).
+
+- **Scala 2.12**.
+Marathon 1.6 is compiled using Scala 2.12. This means that plugins which were compiled using Scala 2.11 will make Marathon fail during startup due to binary incompatibility between Scala 2.11 and 2.12.
+
+## Deprecations
+
+- **/v2/schema route is Deprecated**.
+The /v2/schema route, and JSON Schema definitions, are deprecated in favor of RAML. They will not be kept up-to-date. The endpoint will be disabled in Marathon 1.7.0.
+
+For further details please refer to the changelog.
+
+# Upgrading to 1.5
+
+Release Notes: https://github.com/mesosphere/marathon/releases/tag/v1.5.0
+
+The recommended Mesos version is 1.3.0 or later. Upgrade to Marathon 1.5.x can be performed only from 1.4.x.
+
+## Breaking Changes
```

Comment:
Hm, shouldn't this be part of the release notes? It seems to be an overhead of managing these in addition to the change notes. What do you think?

Reply:
I'm inclined to agree. This stuff (breaking changes) should just be in the release notes.